cli/cmd/proxy/list/command_test.go

```diff
@@ -13,23 +16,25 @@ import (
 )
 
 // getInitializedCommand sets up a command struct for tests.
-func getInitializedCommand(t *testing.T) *ListCommand {
+func getInitializedCommand(t *testing.T) (*ListCommand, *bytes.Buffer) {
 	t.Helper()
 	log := hclog.New(&hclog.LoggerOptions{
 		Name:   "test",
 		Level:  hclog.Debug,
 		Output: os.Stdout,
 	})
 
+	buffer := new(bytes.Buffer)
 	baseCommand := &common.BaseCommand{
 		Log: log,
+		UI:  terminal.NewUI(context.TODO(), buffer),
 	}
 
 	c := &ListCommand{
 		BaseCommand: baseCommand,
 	}
 	c.init()
-	return c
+	return c, buffer
 }
 
 func TestArgumentParsing(t *testing.T) {
```

Comment:
A nit. In other commands, ie control-plane/subcommand/*, these are called flags instead of arguments. Are they called arguments all through the CLI?

Reply:
Makes sense. I'll swap it for flag.